src/lucky_migrator/create_table_statement.cr

```diff
@@ -118,6 +119,41 @@ class LuckyMigrator::CreateTableStatement
     end
   end
 
+  def default_value(type : String.class, default : String, name)
+    if %w[now() current_time current_timestamp].includes?(default.downcase)
+      return " DEFAULT NOW()"
+    end
+    return " DEFAULT '#{default}'"
+  end
+
+  def default_value(type : Time.class, default : (String | Time | Symbol), name)
+    if default.is_a?(Symbol) && default == :now
+      " DEFAULT NOW()"
+    elsif default.is_a?(String) && %w[now() current_timestamp].includes?(default.downcase)
+      " DEFAULT NOW()"
+    elsif default.is_a?(Time)
+      " DEFAULT '#{default.to_utc}'"
+    else
+      raise "Unrecognized default value for #{name}: #{default}. Please use :now, 'now()' or 'current_timestamp'"
+    end
+  end
+
+  def default_value(type : (Int32 | Int64).class, default : (Int32 | Int64), name)
+    if type == Int32 && default > Int32::MAX
+      raise "Cannot set Int64 default for Int32 column '#{name}'. Either set the type to Int64 or change the default value."
+    else
+      " DEFAULT #{default}"
+    end
+  end
+
+  def default_value(type : Bool.class, default : Bool, name)
+    " DEFAULT #{default}"
+  end
+
+  def default_value(type : Float.class, default : (Int32 | Int64 | Float), name)
```

Comment:
I think the default should be restricted to just Float. If the user wants `1` they can use `1.0`